docs/virtual-env.md

```diff
@@ -92,6 +92,7 @@ node --version
 
 - You can use `psql -l` to list available databases, then exit. This will show the credentials to use for `DB_USER` in the 'Owner' column. Update you `.env` file with the following:
 
+       DB_NAME=fala_db
```

Comment:
For the same reasoning we removed it from the example env file I think it doesn't need to be here in the readme, but it's not a big deal.